src/DumbAI.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Random;
+
+/**
+ * Created by Nigel on 6/22/16.
+ */
+public class DumbAI extends AbstractPlayer{
+    private int choose;
+
+    public DumbAI(String name){
+        super (name);
+    }
+
+    @Override
+    public int makeChoice(){
+        int num;
+        Random randomNumber = new Random();
+        num = randomNumber.nextInt(3)+1;
```

Comment:
This could be shortened to `int num = randomNumber.nextInt(3)+1;` which would eliminate line 15.